src/partials/header.html

```diff
@@ -63,7 +63,7 @@
   </ul>
 </div>
 <header class="header">
-  <div class="container-modificator container">
+  <div class="container-modificator containerr">
```

Comment:
si esta bien escrito =¿?

Reply:
Si, esque tu en tu sass de how it's made tambien usaste el container, lo que pasa es que yo no use el container para hacer mi sass de header entonces dañabas inconscientemente mis estilos, por eso le puse una doble rr para que no use tus cambios.